pkgs/top-level/all-packages.nix

```diff
@@ -5484,6 +5484,8 @@ with pkgs;
 
   gdu = callPackage ../tools/system/gdu { };
 
+  geticons = callPackage ../by-name/ge/geticons/package.nix { };
+
```

Comment:
```suggestion
```
Packages found in the named-based structure do not need to be explicitly added to the `top-level/all-packages.nix` file unless they require overriding the default value of an implicit attribute.